package.json

```diff
@@ -36,9 +36,9 @@
   },
   "scripts": {
     "test": "npm run test:client && npm run test:common && npm run test:server",
-    "test:common": "node_modules/.bin/mocha --require ts-node/register --require ignore-styles src/common/**/*.mocha.{ts,tsx}",
```

Comment:
We can use the short form.

According to https://docs.npmjs.com/cli/run-script
> In addition to the shell's pre-existing PATH, npm run adds node_modules/.bin to the PATH provided to scripts. Any binaries provided by locally-installed dependencies can be used without the node_modules/.bin prefix.

The docs also specifically encourage to use the short form instead of the relative one.